AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/GraphQL/GraphQLWithIAMIntegrationTests/GraphQLWithIAMIntegrationTests.swift

```diff
@@ -13,88 +13,7 @@ import AWSAPICategoryPlugin
-        * Do you want to configure advanced settings for the GraphQL API `No, I am done.`
-        * Do you have an annotated GraphQL schema? `No`
-        * Do you want a guided schema creation? `Yes`
-        * What best describes your project: `Objects with fine-grained access control (e.g., a project management app
-           with owner-based authorization)`
-        * Do you want to edit the schema now? `No`
-
-    3. Add Auth `amplify add auth`
-    Using service: Cognito, provided by: awscloudformation
-
-     The current configured provider is Amazon Cognito.
-
-     Do you want to use the default authentication and security configuration? `Default configuration`
-     Warning: you will not be able to edit these selections.
-     How do you want users to be able to sign in? Username
-     Do you want to configure advanced settings? `No, I am done.`
-    Successfully added resource appsyncsample6b51ebcc locally
-
-    4. `amplify push`
-       * Do you want to generate code for your newly created GraphQL API `Yes`
-       * Enter the file name pattern of graphql queries, mutations and subscriptions `graphql/**/*.graphql`
-       * Do you want to generate/update all possible GraphQL operations - queries, mutations and subscriptions `Yes`
-       * Enter maximum statement depth [increase from default if your schema is deeply nested] `2`
-       * Enter the file name for the generated code `API.swift`
-       * GraphQL endpoint: `https://szc4yxxxxxxxxxxqaaiwoqe.appsync-api.us-east-1.amazonaws.com/graphql`
-
-    5. Update the IAMPolicy to allow operations on the GraphQL service
-     TODO: Fix these instructions
-    */
-
-    /*
-     we need to update AmplifyTestApp to use the right cognito which is set up with guest access
-
-     this is working only because iam policy is set up correctly.
-
-     {
-     "UserAgent": "aws-amplify/cli",
-     "Version": "0.1.0",
-     "IdentityManager": {
-         "Default": {}
-     },
-     "AppSync": {
-         "Default": { // this is `todoGraphQLWithIAMWithGuestAccess`
-             "ApiUrl": "https://fsdfgjw5ojdanhivobrnmw54s4.appsync-api.us-east-1.amazonaws.com/graphql",
-             "Region": "us-east-1",
-             "AuthMode": "AWS_IAM",
-             "ClientDatabasePrefix": "api3_AWS_IAM"
-         }
-     },
-     "CredentialsProvider": {
-         "CognitoIdentity": {
-             "Default": {
-                 "PoolId": "us-east-1:35d7a505-675c-440e-969f-82d0773b586b",
-                 "Region": "us-east-1"
-             }
-         }
-     },
-     "CognitoUserPool": {
-         "Default": {
-             "PoolId": "us-east-1_PdSfW6IgE",
-             "AppClientId": "5gf44rm3rv8ar0o9spcuols1g4",
-             "AppClientSecret": "11343jpcqhj7dqvacjkq5hlrcrlauqka56jg7lhi00vq1ra1b57i",
-             "Region": "us-east-1"
-         }
-     }
-     */
-
-    let user1 = "storageUser1@testing.com"
+    let user1 = "user1@GraphQLWithIAMIntegrationTests.com"
     let password = "Abc123@@!!"
```

Comment:
wait, that's the password i use for all my accounts.... :P

Reply:
haha yeah, we should discuss some alternatives to how to set up tests which require users sign in. let's continue the conversation over in the storage PR (that pr builds on top of this one and i don't want to have to rebase things if any major changes)